runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -824,6 +824,8 @@ ClientSessionHT::findOrCreateClientSession(uint64_t clientUID, uint32_t seqNo, b
          *newSessionWasCreated = true;
          if (TR::Options::getVerboseOption(TR_VerboseJITServer))
             TR_VerboseLog::writeLineLocked(TR_Vlog_JITServer, "Server allocated data for a new clientUID %llu", (unsigned long long)clientUID);
+         else if (TR::Options::getVerboseOption(TR_VerboseJITServerConns))
```

Comment:
As above, we may not want to exclude one message because we got the other. If we use some scripts to grep through the verbose log we may miss some messages.